torrents/management/commands/add_snatched_by_tracker.py

```diff
@@ -0,0 +1,188 @@
+from bencode import bdecode
+import os
+import shutil
+
+from django.core.management import BaseCommand
+
+from torrents.download_locations import format_download_path_pattern
+from torrents.models import TorrentInfo, TorrentFile, Realm, Torrent
+from torrents.add_torrent import add_torrent_from_tracker, fetch_torrent
+from torrents.exceptions import RealmNotFoundException
+
+from trackers.registry import TrackerRegistry
+
+class ImportedTorrentMover():
+    # store_files_hook for add_missing_from_tracker
```

Comment:
Do you know how to use docstrings? The comments are good, it's just better when they're in a standard way. It's best to look it up online, but something like
```
class ImportedTorrentMover:
    """
    Some description
    on how the class works
    """

    def __init__(self, arg):
        """You can put more stuff in here, including describing the arguments."""
````